.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: Pull request
+on:
+  pull_request:
+    types: [opened, closed, reopened, ready_for_review, review_requested]
+
+jobs:
+
+  notifications:
+    name: Telegram notifications
+    runs-on: ubuntu-latest
+    steps:
+      - name: Send telegram message on pull request to review
+        if: github.event.pull_request.merged == false && github.event.action != 'closed'
+        uses: appleboy/telegram-action@master
+        with:
+          to: ${{ secrets.TELEGRAM_TO }}
+          token: ${{ secrets.TELEGRAM_TOKEN }}
+          format: markdown
+          message: |
+            ${{ github.repository }} repository
+
+            ${{ github.actor }} created new PR: [#${{ github.event.number }}](${{ github.event.pull_request.html_url }})
+
+            Title: ${{ github.event.pull_request.title }}
+      - name: Send telegram message on pull request merged to main
+        if: github.event.pull_request.merged == true
+        uses: appleboy/telegram-action@master
+        with:
+          to: ${{ secrets.TELEGRAM_TO }}
```

Comment:
@santisiri could you help with this? I think only have permissions to set these secrets.